### PR TITLE
Implemented the post scenarios for all 5 features

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

File: src/test/resources/features/contracts.feature [hunk 1]
```diff
@@ -14,10 +14,19 @@ Feature: Contracts
...
+#    Given I build a "POST" request for a Contract
+#    When I create Contract body with start date "25/06/2021"
+#    And I execute the post Contract request on "/Contract" endpoint
+#    Then The response status code should be "CREATED" on post Contract request
+    Given I build a "POST" request
+    When I create body with parameters
+      | object | Contract |
+      | accountId | 0015e00000B0PtfAAF |
+      | status | Draft |
+      | startDate | 28/06/2021 |
```
Comment (JessickaMoya) [inline src/test/resources/features/contracts.feature]: Change format of date. It should be "YYYY-MM-DD"

File: src/test/resources/features/orders.feature [hunk 2]
```diff
@@ -14,10 +14,19 @@ Feature: Orders
...
+#    Given I build a "POST" request for a Order
+#    When I create Order body with start date "29/06/2021"
+#    And I execute the post Order request on "/Order" endpoint
+#    Then The response status code should be "CREATED" on post Order request
+    Given I build a "POST" request
+    When I create body with parameters
+      | object | Order |
+      | accountId | 0015e00000B0PtfAAF |
+      | status | Draft |
+      | effectiveDate | 01/07/2021 |
```
Comment (JessickaMoya) [inline src/test/resources/features/orders.feature]: Change format of date. It should be "YYYY-MM-DD"

File: src/test/java/rest/salesforce/org/steps/ApiSteps.java [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+    @Given("I build a {string} request")
+    public void iBuildARequest(final String apiMethod) {
+        requestBuilder.addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addMethod(ApiMethod.valueOf(apiMethod));
+    }
+
+    @When("^I create body with parameters$")
+    public void iCreateBodyWithParameters(final Map<String, String> entry) throws JsonProcessingException, ParseException {
```
Comment (cristianfchq) [inline src/test/java/rest/salesforce/org/steps/ApiSteps.java:161]: this method is long maybe you can divide in submethods

File: src/test/resources/features/accounts.feature [hunk 4]
```diff
@@ -2,32 +2,53 @@ Feature: Accounts
   Description
   @CreateAndDeleteAccount
   Scenario: Get a single Account
-    Given I build a "GET" request for a single Account
-    When I execute the get single Account request on "/Account/{AccountId}" endpoint
-    Then The response status code should be "OK" on get single Account request
+#    Given I build a "GET" request for a single Account
```
Comment (cristianfchq) [inline src/test/resources/features/accounts.feature:5]: This comments I think that is not necesary

File: src/test/java/rest/salesforce/org/steps/ApiSteps.java (hunk 3 above)
Comment (Raymundogs) [reply src/test/java/rest/salesforce/org/steps/ApiSteps.java:161]: I will  change the implementation of this step, but due to the lack of time I will make it on my next PR.

**Review by JessickaMoya — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**